### DISCUSSION TITLE
Extend matchers in browser mode

### DISCUSSION THREAD

**Discussion by edoardocavazza** [vitest-dev/vitest#3269] [Q&A] [answered]
Hi!

I'm trying to globally extend matchers in browser mode, but I can't find where to do it. If I understand correctly, `globalSetup` is meant to be used for "node-side" things. How can I load a client-side library before running tests?

**Comment by sheremet-va** [comment, marked as answer]
the same as you would for node by using setupFiles

**Reply by edoardocavazza** [reply]
Ups, thank you. I had missed it 🙏
